### ISSUE TITLE
Import 08eedf7f20: Put secondary camera back into deferred, rather than forward, rendering (by Caitlyn)

### ISSUE THREAD

**Issue by daleglass** [overte-org/tivolicloud#228]
This is an automated proposal to look at a commit made by Caitlyn and import it into [Overte](https://github.com/overte-org/overte)

| **Commit:** |  08eedf7f20539360d6222354e11b5f1eac487b18 |
| --- | --- |
| **Author:** | Caitlyn |
| **Date:** | Sun, 08 Mar 2020 13:32 |

```
Put secondary camera back into deferred, rather than forward, rendering


```

**Stats:**

| **Filename** | Stats | Lines | Added | Removed | Lines in blame |
| --- |  --- | --- | --- | --- | --- |
| interface/src/graphics/GraphicsEngine.cpp |  | 4 | 2 | 2 | ⚠ 0
| 1 files | - | 4 | 2 | 2 | 0 |

To work on this, please assign the issue to yourself, then look at the commit and decide whether this would be a good addition to Overte.

* If the commit is useful, tag it with "Tivoli: Keep", and keep it open until it's merged.
* If the commit is not useful, tag it with "Tivoli: Discard", and close it.
* If the commit is not useful right now, but might be later, tag it with "Tivoli: Maybe later", and close it.
* If it's hard to decide, tag it with "Tivoli: Discuss", and keep it open.

Useful commits should be submitted as a [PR against Overte](https://github.com/overte-org/overte/pulls). Tag this issue in PR, so that it's automatically closed once the PR is merged.

You can cherry-pick this issue with this command:

    git cherry-pick 08eedf7f20539360d6222354e11b5f1eac487b18

**Comment by daleglass** [comment]
Trivial change, disables stereo and deferred rendering. Benefit seems doubtful.
